src/config/baselines.tsx

```diff
@@ -0,0 +1,55 @@
+import nsoDisabled from '../data/nso/NSO_Disabled_Admin1_Total.json';
+import nsoHerders from '../data/nso/NSO_Herder_HHs_Admin2.json';
+import nsoHerdsize from '../data/nso/NSO_Herd_Size_Admin1_LT_200.json';
+import nsoChild from '../data/nso/NSO_Child_U5_Admin2.json';
+import nsoLivestock from '../data/nso/NSO_Livestock_Count_ths_Admin2.json';
+import nsoElderly from '../data/nso/NSO_Single_Elderly_Admin1_Total.json';
+import nsoPoverty from '../data/nso/NSO_Poverty_Headcount_Admin1.json';
+import nsoPop from '../data/nso/NSO_Population_Admin2_Total.json';
+
+export interface NsoData {
+  TBL_ID: string;
+  Period: string;
+  CODE?: string;
+  SCR_MN: string;
+  SCR_ENG: string;
+  CODE1?: string;
+  CODE2?: null;
+  SCR_MN1: string;
+  SCR_ENG1: string;
+  SCR_MN2?: null;
```

Comment:
Same with these, are these actually `null` typed?

Reply:
Looks like it. I am getting an error if I say string.
Should I say `string | null`?

Reply:
Ya, that's the appropriate type. Javascript handles `null` and `undefined` differently - the `?` only covers `undefined`.

It's actually somewhat intentional - `undefined` is intended to signify that nothing was, well, defined, whereas `null` is intended to an explicit signal that the value is null'ed out. In vanilla JS it gets confused because the type-casting with boolean-ish values is so imprecise that it's easy to get sloppy with the distinction.